### DISCUSSION TITLE
Fedora 41 support

### DISCUSSION THREAD

**Discussion by dehnes** [wezterm/wezterm#6325] [General]
Hi, on the download page it says Fedora 40 is supported by nightly build. Since Fedora 41 is still in beta but will hit on 29 Oct can we expect to have officially support soon? Thanks in advance!

**Comment by olejorgenb** [comment]
I think it's just the docs which is outdated. https://copr.fedorainfracloud.org/coprs/wezfurlong/wezterm-nightly/ list support for 41
